### PR TITLE
MPI jobs

### PR DESCRIPTION
**Pull request by shaohao99**
Add recipes for MPI jobs.

### CODE REVIEW

**Comment by shaohao99** [comment]
Preview
https://mit-orcd.github.io/orcd-docs-previews/PR/PR38/recipes/mpi/

File: docs/recipes/mpi.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+#!/bin/bash
+#SBATCH -p sched_mit_hill
+#SBATCH -t 30
+#SBATCH -N 2
+#SBATCH --ntasks-per-node=2
+#SBATCH -c 8
+#SBATCH --mem=0
+
+module load gcc/6.2.0 openmpi/3.0.4
+export OMP_NUMB_THREADS=$SLURM_CPUS_PER_TASK
```
Comment (lmilechin) [inline docs/recipes/mpi.md:192]: Typo: OMP_NUMB_THREADS -> OMP_NUM_THREADS

File: docs/recipes/mpi.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+```
+#!/bin/bash
+#SBATCH -p sched_mit_hill
+#SBATCH -t 30
+#SBATCH -N 1
+#SBATCH -n 16
+#SBATCH --mem=10GB
+
+module load gcc/6.2.0 openmpi/3.0.4
+export OMP_NUMB_THREADS=8
```
Comment (lmilechin) [inline docs/recipes/mpi.md:209]: Typo: OMP_NUMB_THREADS -> OMP_NUM_THREADS

File: docs/recipes/mpi.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+If replacing `--ntasks-per-node=8` with `-n 16` in the above script, the job will request 16 cores on 2 nodes, but it is not always the case that there are 8 cores per node. For example, there may be 7 cores on one node and 9 cores on another, or 1 core on one node and 15 cores on another, etc, depending on the current available resources on the cluster. 
+
+
+## Computing resources for MPI jobs
+
+To get a better idea on how many nodes, cores and memory should be requested, users need to consider the following two questions. 
+
+First, what resources are available on the cluster? Use this command to check node and job info on the cluster, including the constraint associated with OS (`%f`), the nubmer of CPU cores (`%c`), the memory size (`%m`), the wall time limit (`%l`), and the current usage status (`%t`). 
+```
+ sinfo -N -p sched_mit_hill,newnodes,sched_any,sched_mit_orcd -o %f,%c,%m,%l,%t |grep -v drain
```
Comment (lmilechin) [inline docs/recipes/mpi.md:129]: I suggest leaving out the -p flag, by default users will only see partitions they can submit jobs to, and adding %P to the output so users know which partition to select:
` sinfo -N -o %P,%f,%c,%m,%l,%t |grep -v drain`

File: docs/recipes/relion.md
Comment (lmilechin) [inline docs/recipes/relion.md:1]: Did you intend to include RELION in this PR?

**Review by lmilechin — CHANGES_REQUESTED** [review]

File: docs/recipes/relion.md
Comment (shaohao99) [reply docs/recipes/relion.md:1]: Yes, RELION is good to go too.

Comment (lmilechin) [reply docs/recipes/relion.md:1]: Sounds good, I'll read through that one as well.

File: docs/recipes/relion.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+# Installing and Using RELION
+
+RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a software package that employs an empirical Bayesian approach for electron cryo-microscopy (cryo-EM) structure determination. 
+
+## RELION on Satori 
+
+This recipe is for building and using RELION on x86 nodes on Satori. It is different from working on IBM power9 nodes on Satori. 
+
+> Note: The x86 nodes are available to some labs only.
```
Comment (lmilechin) [inline docs/recipes/relion.md:17]: Did you want to use the [admonition syntax](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) for the Notes, as you did in the MPI pages? This one:

!!! Note
    The x86 nodes are available to some labs only.

File: docs/recipes/relion.md
Comment (lmilechin) [reply docs/recipes/relion.md:1]: Looks good, just the one comment/suggestion about the notes.

**Comment by shaohao99** [comment]
I have modified both pages of MPI and RELION, and pushed it. I think it is good to go.

**Comment by shaohao99** [comment]
The MPI4py for EO is added to the page of MPI4py.

**Pull request merged**